ra/proto/ra.proto

```diff
@@ -96,12 +96,15 @@ message AdministrativelyRevokeCertificateRequest {
 }
 
 message NewOrderRequest {
-  // Next unused field number: 7
+  // Next unused field number: 8
   int64 registrationID = 1;
   repeated string dnsNames = 2;
+  string certificateProfileName = 5;
+  // Replaces is the ARI certificate Id that this order replaces.
+  string replaces = 7;
+  // ReplacesSerial is the serial number of the certificate that this order replaces.
   string replacesSerial = 3;
```

Comment:
Should this be commented as deprecated, or TODO: remove? It's now redundant with the latter half of the new `replaces` field.

Reply:
Color me convinced!